### PR TITLE
Home focus

### PR DESCRIPTION
**Pull request by SRUJANA-PENUGONDA13**
Refactored code and replaced existing weather api with open map weather api

### CODE REVIEW

File: src/components/Home.js [hunk 1]
```diff
@@ -1,36 +1,13 @@
...
-    if (navigator.geolocation) {
-      navigator.geolocation.getCurrentPosition(
-        (position) => {
-          let latitude, longitude;
-          latitude = position.coords.latitude;
-          longitude = position.coords.longitude;
-          getWeatherData(latitude, longitude);
-        },
-        (error) => {
-          console.log("Error: ", error);
```
Comment (NeerajVodala) [inline src/components/Home.js:29]: console.error would be semantically correct here

File: src/css/App.css [hunk 2]
```diff
@@ -50,24 +50,28 @@
...
   margin: auto;
   margin-right: 0px;
+  line-height: 10px;
   font-size: var(--text-medium);
 }
 
+.weatherIcon {
+  height: 80px;
+  width: 80px;
+}
```
Comment (NeerajVodala) [inline src/css/App.css]: use relative units where ever possible

**Review by NeerajVodala — APPROVED** [review]
Try to use console.error instead of .log if you want to show error.
Good that you have used try catch to handle errors.
Rest everything looks good.

File: src/utils/greeting.js [hunk 3]
```diff
@@ -2,11 +2,11 @@ const getGreeting = (ctime) => {
   const userName = localStorage.getItem("user-name");
   let greeting = "";
   if (ctime !== "" && ctime !== undefined) {
+    let hours = ctime.slice(0, 2).trim().replace(/:$/g, "");
+    let noon = ["12", "01", "02", "03", "04", "05", "1", "2", "3", "4", "5"];
```
Comment (LordUttam) [inline src/utils/greeting.js]: This looks overcomplicated. If you are unsure of what time format might come here just control it at source with locales argument when using `toLocaleTimeString()` 
Example:
// US English uses 12-hour time with AM/PM
console.log(date.toLocaleTimeString('en-US'));
// → "7:00:00 PM"

// British English uses 24-hour time without AM/PM
console.log(date.toLocaleTimeString('en-GB'));
// → "03:00:00"

**Review by LordUttam — COMMENTED** [review]
For greeting good afternoon and everything why not use conditions like `hours<16 && hours>12` ? 
Would also suggest to only show hours and minutes else update the second every second. Try making the time and greeting components separate so it will be easier to understand and control.

File: src/components/Home.js (hunk 1 above)
Comment (SRUJANA-PENUGONDA13) [reply src/components/Home.js:29]: Updated

File: src/css/App.css (hunk 2 above)
Comment (SRUJANA-PENUGONDA13) [reply src/css/App.css]: Updated

File: src/utils/greeting.js (hunk 3 above)
Comment (SRUJANA-PENUGONDA13) [reply src/utils/greeting.js]: Updated

**Comment by SRUJANA-PENUGONDA13** [comment]
Thank You @LordUttam @NeerajVodala for reviewing

**Pull request merged**